app/models/concerns/sufia/ability.rb

```diff
@@ -52,6 +52,7 @@ def editor_abilities
 
       can :create, TinymceAsset
       can [:create, :update], ContentBlock
+      can :edit, ::SolrDocument
```

Comment:
I think we're trying to get away from having to do overrides.  I'm strongly advocating putting this in the rights metadata. Someone should be able to understand who has access even if Hydra goes away